setup.py

```diff
@@ -23,7 +23,7 @@
 
 setup(
     name="splicemachine",
-    version="2.7.0",
+    version="2.7.0dev0",
```

Comment:
is this on purpose

Reply:
yup - in line with the new stuff we're doing for ml-workflow. Since master is really things that are "in development" we don't want to accidentally squash the official package versions. So I made it dev0